eth/stagedsync/exec3_parallel.go

```diff
@@ -459,7 +459,7 @@ func (pe *parallelExecutor) run(ctx context.Context, maxTxNum uint64, logger log
 	pe.in = state.NewQueueWithRetry(100_000)
 
 	pe.execWorkers, _, pe.rws, pe.stopWorkers, pe.waitWorkers = exec3.NewWorkersPool(
-		pe.RWMutex.RLocker(), pe.accumulator, logger, ctx, true, pe.cfg.db, pe.rs, pe.in,
+		pe.RWMutex.RLocker(), pe.accumulator, logger, nil, ctx, true, pe.cfg.db, pe.rs, pe.in,
```

Comment:
Is it OK that hooks are `nil` in parallel execution?

Reply:
Should be ok, we don't pass any tracers in the previous version, we just create a call tracer in `NewWorker` which we do here as well.